src/javascript/binary/base.js

```diff
@@ -222,3 +222,13 @@ if (!/backoffice/.test(document.URL)) { // exclude BO
         LocalStore.set('active_loginid', match);
     });
 }
+
+
+//this method is to prevent caching on sensitive data
+pjax_config_page("new_account/real", function(){
```

Comment:
the current html form is generated by a perl module, which does not seems to support tag addition, to make it work require some changes in the perl module which I have no knowledge of, current way of doing it should not possess any security concerns, 
but you're right, it is confusing, so do you want me to fiddle with the perl script or you are OK for this at the moment?